modules/api/src/main/scala/fi/spectrum/api/v1/services/PriceTracking.scala

```diff
@@ -0,0 +1,139 @@
+package fi.spectrum.api.v1.services
+
+import cats.syntax.traverse._
+import cats.{Functor, Monad, Parallel}
+import derevo.derive
+import fi.spectrum.api.currencies.UsdUnits
+import fi.spectrum.api.db.models.amm.{PoolSnapshot, PoolVolumeSnapshot}
+import fi.spectrum.api.db.repositories.Pools
+import fi.spectrum.api.modules.PriceSolver.FiatPriceSolver
+import fi.spectrum.api.services._
+import fi.spectrum.api.v1.endpoints.models.TimeWindow
+import fi.spectrum.api.v1.endpoints.monthMillis
+import fi.spectrum.api.v1.models.amm._
+import fi.spectrum.graphite.Metrics
+import tofu.doobie.transactor.Txr
+import tofu.higherKind.Mid
+import tofu.higherKind.derived.representableK
+import tofu.logging.{Logging, Logs}
+import tofu.syntax.doobie.txr._
+import tofu.syntax.logging._
+import tofu.syntax.monadic._
+import tofu.syntax.time.now.millis
+import tofu.time.Clock
+
+@derive(representableK)
+trait PriceTracking[F[_]] {
+
+  def getMarkets(window: TimeWindow): F[List[AmmMarketSummary]]
+}
+
+object PriceTracking {
+
+  def make[I[_]: Functor, F[_]: Monad: Clock: Parallel, D[_]: Monad](implicit
+    txr: Txr[F, D],
+    pools: Pools[D],
+    tokens: VerifiedTokens[F],
+    snapshots: Snapshots[F],
+    solver: FiatPriceSolver[F],
+    metrics: Metrics[F],
+    logs: Logs[I, F]
+  ): I[PriceTracking[F]] =
+    logs
+      .forService[PriceTracking[F]]
+      .map(implicit __ => new PriceTrackingMetrics[F] attach (new Tracing[F] attach new Live[F, D]))
+
+  final class Live[F[_]: Monad: Clock: Parallel: Logging, D[_]: Monad](implicit
+    txr: Txr[F, D],
+    pools: Pools[D],
+    tokens: VerifiedTokens[F],
+    solver: FiatPriceSolver[F],
+    snapshots: Snapshots[F]
+  ) extends PriceTracking[F] {
+
+    def getMarkets(window: TimeWindow): F[List[AmmMarketSummary]] =
```

Comment:
And we can also remove original method from AMM routes and add it to price tracking routes

Reply:
And  routes will be `/markets/verified` and `/markets`